openstack/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG - openstack
 
+## 1.6.0 / 2018-11-30
+
+* [Added] Adds ability to Trace "check" function with DD APM. See [#2079](https://github.com/DataDog/integrations-core/pull/2079).
```

Comment:
This was reverted here - https://github.com/DataDog/integrations-core/pull/2350

#2079 has some changes that didn't get reverted so I don't want to change the changelog label there. Can we manually remove this?